### DISCUSSION TITLE
Space is not present between words in extracted table dataframe.

### DISCUSSION THREAD

**Discussion by AyushaKadi** [jsvine/pdfplumber#1194] [Ask for help with specific PDFs]
Hi I am using the pdfplumber library and  'extract_vertical_lines' and 'extract_horizontal_lines' as table extract features to capture transactions spanning between multiple lines. The tables are perfectly being captured and so are the transactions. 
The only issue is spacing between the words captured
<img width="352" alt="Screenshot 2024-08-30 011700" src="https://github.com/user-attachments/assets/97ed02d7-cb8e-431b-b7d6-dad3d13db57c">
 I am unable to debug why the words which have space in the pdf are not generated as same in the text or dataframe.
 Example- PointOf SaleWithdrawal PUBLIXSUPERMA 6901TAFT ST HOLLYWOOD FLUS
 Expected- Point Of Sale Withdrawal PUBLIX SUPER MA 6901 TAFT ST HOLLYWOOD FLUS
Can this be resolved by any way? I have tried with tolerances and explicit codes for managing tolerances and space too.

v

**Comment by jsvine** [comment]
Hi @AyushaKadi, have you tried adjusting the `text_x_tolerance` parameter? (It's the table-setting equivalent to `x_tolerance` described in the documentation related to text extraction.)

**Reply by AyushaKadi** [reply]
No. I haven't tried it. Will definitely try it out. Thank you for your feedback.
